docs/Archive/cubic-X11GUI.md

```diff
@@ -1,18 +1,18 @@
 ---
 layout: default
 title: CUBIC-X11GUI
-parent: archive
+parent: Archived Documentation
```

Comment:
are you able to rename the folder? I am not

Reply:
I can give it a try

Reply:
I'm fine with `Archive` at this point though. As long as all of the archived files are in the same folder.

Reply:
So they appear to be under the same heading, but there are two different folders

Reply:
Should be good now.